src/utils/parse-query.js

```diff
@@ -35,4 +36,36 @@ const parseQuery = (query) => {
   return parsedQuery;
 };
 
-export { parseQuery };
+/**
+ * Generate a WHERE clause based on parsed filters
+ *
+ * @param {object} parsedFilters Parsed filter parameters
+ * @returns {string} WHERE clause string
+ */
+const generateWhereClause = (parsedFilters) => {
```

Comment:
Should this function be placed in `parseQuery` file? I feel more like this is a DAO-related function.

Reply:
as filters are already made inside parsed query so I thought let's put this clause generation here as well, because it handles the query parsing and now building it based on the parsed filters.

Reply:
Hmm, it doesn't sound right to me. The purpose of this function is mainly for generating the where clause for your SQL and shouldn't be used by any other function that imports this file, so I'm not seeing a reason why we should put the logic here.

Reply:
sure, makes sense to me. moving it.